### DISCUSSION TITLE
Playbook Alert/Hunt showing incorrect timestamp

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#2069] [Unsupported Versions] [answered]
Alert/Hunt show some default timestamp instead of the @ timestamp field for Playbook rules:

0001-01-01 12:09:21.000 AM +00:09 | Renamed PowerShell | critical |   |   |   |   |   | 961ff68ae | process_creation |  
@timestamp | 2020-11-24T12:11:13
event.dataset | alert
event.severity | 4
event.severity_label | critical
event_data.@timestamp | 2020-11-24T12:10:52.622Z

0001-01-01 12:09:21.000 AM +00:09 | Highly Relevant Renamed Binary | high |   |   |   |   |   | 3f79392fa | process_creation |  
@timestamp | 2020-11-24T12:12:41
event.dataset | alert
event.module | playbook
event.severity | 3
event.severity_label | high
event_data.@timestamp | 2020-11-24T12:10:52.622Z

**Comment by weslambert** [comment]
Hi @0nnyx , 

Thanks for reporting! We probably need to adjust the parsing here to pull `event_data.timestamp` as `@timestamp` and record the value of `@timestamp` as `ingest.timestamp` (because its merely referring to to when the event was ingested into Elasticsearch), similar to how we do with other datasets.  @defensivedepth, any thoughts?

**Reply by 0nnyx** [reply]
Hi  @defensivedepth, just want to highlight this default timestamp is still present on the latest 2.3.21

**Comment by dougburks** [comment, marked as answer]
You can track this issue here:
https://github.com/Security-Onion-Solutions/securityonion/issues/2071
